frontend/src/components/v1/DraftPresenceIndicator.css

```
@@ -0,0 +1,106 @@
+.draft-presence {
+  --draft-bg: color-mix(in oklab, var(--surface-elevated, rgba(255, 255, 255, 0.08)) 92%, transparent);
+  --draft-border: color-mix(in oklab, var(--glass-border, rgba(255, 255, 255, 0.16)) 86%, transparent);
+  --draft-text: var(--text-main, #f5f7fb);
+  --draft-text-muted: var(--text-secondary, #a8b5c8);
+  display: inline-flex;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)